### PR TITLE
run go mod tidy to remove unused packages

### PR DESCRIPTION
**Pull request by grosser**

### CODE REVIEW

**Comment by grosser** [comment]
@aksgithub

File: .travis.yml [hunk 1]
```diff
@@ -5,5 +5,8 @@ branches:
 go: "1.12"
 env:
 - GO111MODULE=on
-# NOTE: `go fmt` does not work, see https://go-review.googlesource.com/c/go/+/153459/
-script: go build cmd/remediator/app.go && [ -z "`ls */**/*.go | xargs -n1 go fmt`" ]
+script: |
+  set -ex
+  go build -o .build/remediator cmd/remediator/app.go
+  [ -z "`ls */**/*.go | xargs -n1 go fmt`" ] # `go fmt` does not work, see https://go-review.googlesource.com/c/go/+/153459/
```
Comment (uthark) [inline .travis.yml]: But the issue was fixed in Dec 2018 in 1.12. https://github.com/golang/go/issues/28011

Comment (grosser) [reply .travis.yml]: 🤷‍♂ 
```
1.12.10
go fmt
can't load package: package github.com/aksgithub/kube_remediator: unknown import path "github.com/aksgithub/kube_remediator": cannot find module providing package github.com/aksgithub/kube_remediator
```

Comment (uthark) [reply .travis.yml]: `go fmt ./...` works perfectly.

Comment (grosser) [reply .travis.yml]: updated

**Review by uthark — APPROVED** [review]

**Pull request merged**